src/main/scripts/jbang

```diff
@@ -133,11 +133,22 @@ execute_jbang() {
   fi
   
   if [ $err -eq 255 ]; then
-    eval "exec $output"
+    if [[ "$os" == "windows" ]] && [[ "$JBANG_WIN_UTF8" != "false" ]]; then
+      bash -c "$output"
```

Comment:
why bash not exec here?

Reply:
We need the command to return to the curent shell , to be able to restore the code page setting at the end of the script.

eval "exec ..." does not return.